index.html

```diff
@@ -0,0 +1,44 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+  </head>
+  <body>
+    <div class="container">
+      <!-- navbar -->
+      <nav class="navbar">
+
+        <div class="logo">
+          <img src="my_logo.png">
+        </div>
+
+        <div class="union">
+          <img src="union.png">
+        </div>
+
+      </nav>
+    <!-- introduction -->
+      <section class="introduction">
+        <h1 class="header">
+          I'm Aleem <br>
+          Glad to see <br>
+          you!
+        </h1>
+        <p class="intro">
+          I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <a href="" class="button">LET'S CONNECT</a>
+        <ul class="profiles">
+          <li><a href=""><img src="twitter.png"></a></li>
+          <li><a href=""><img src="linkedin.png"></a></li>
+          <li><a href=""><img src="github.png"></a></li>
```

Comment:
- [ Required ] As requested by the previous reviewer please add the links to your social media pages, please note that if the `href` attribute is empty these links will not lead a person who wants to connect with you to your social media profiles. Kindly rectify this